package.json

```diff
@@ -1,17 +1,21 @@
 {
-  "name": "node-api",
-  "main": "server.js",
-  "dependencies": {
-    "body-parser": "~1.0.1",
-    "express": "~4.0.0",
-    "http-errors": "^1.6.2",
-    "moment": "^2.18.1",
-    "mongoose": "^4.10.8",
-    "mongoose-unique-validator": "^1.0.5",
-    "redis": "^2.7.1",
-    "request": "^2.81.0"
-  },
-  "engines": {
-    "node": "8.2.1"
-  }
+    "name": "node-api",
+    "main": "server/server.js",
+    "dependencies": {
+        "body-parser": "~1.0.1",
+        "express": "~4.8.0",
+        "http-errors": "^1.6.2",
+        "moment": "^2.18.1",
+        "mongoose": "^4.10.8",
+        "mongoose-unique-validator": "^1.0.5",
+        "redis": "^2.7.1",
+        "request": "^2.81.0"
+    },
+    "engines": {
+        "node": "8.2.1"
+    },
+    "scripts": {
+        "start": "node server/server.js",
+        "heroku-postbuild": "cd web && npm install && npm install --only=dev --no-shrinkwrap && npm run build"
```

Comment:
What is this for?

Reply:
Oh is this to install dependencies for the web project?

Reply:
See here for an example from an official heroku guy: https://github.com/mars/heroku-cra-node

It's to get Heroku to run the express backend and a react web service using create-react-app for minimal configuration.

Reply:
https://daveceddia.com/create-react-app-express-production/

Reply:
Oh interesting. Because you need to build the react client AFTER the server is built

Reply:
Right. Heroku is building and deploying the Express Node app, but that doesn't compile/build/webpack the source React code into /web/build, from whence the web stuff is actually served